Compiler.coffee

```diff
@@ -70,8 +70,12 @@ class Compiler
         props.className = className
 
       React.cloneElement child, props,
-        React.Children.map child.props.children, (child) =>
-          @traverseChild child, parentClass
+        if React.Children.count(child.props.children) is 1
```

Comment:
Actually, we can just check `Array.isArray(child.props.children)` and call `@traverseChild child.props.children, parentClass` if statement is false